src/tools/perf/lib/ucp_tests.cc

```diff
@@ -504,6 +504,20 @@ class ucp_perf_test_runner {
         ucp_worker_fence(m_perf.ucp.worker);
     }
 
+    void reset_buffers(const void *src, size_t offset, size_t count)
+    {
+        if (CMD == UCX_PERF_CMD_PUT) {
+            m_perf.allocator->memcpy((uint8_t *)m_perf.recv_buffer + offset,
+                                     m_perf.allocator->mem_type,
+                                     src, UCS_MEMORY_TYPE_HOST,
+                                     count);
+            m_perf.allocator->memcpy((uint8_t *)m_perf.send_buffer + offset,
```

Comment:
can you pls use UCS_PTR_BYTE_OFFSET(m_perf.recv_buffer, offset)
same for sender

Reply:
Done with the 2nd commit